fastlane/Fastfile

```diff
@@ -181,6 +176,13 @@ REPOSITORY_NAME = 'WordPress-Android'
   desc 'Trigger a release build for a given app after code freeze'
   lane :complete_code_freeze do |options|
     android_completecodefreeze_prechecks(skip_confirm: options[:skip_confirm])
+
+    localize_libraries()
+    update_frozen_strings_for_translation()
+
+    ensure_git_status_clean()
+    push_to_git_remote
```

Comment:
> This also allowed us to only make a single git push just before we trigger the CI build, instead of two previously

A nice improvement 👍 

Even when this repo will be migrated to Buildkite, thus not starting then cancelling a build resulting in a confusing red x in the commit status, in the context of this kind of automated workflow, it's good to avoid pushing when we know for a fact we'll push something newer right afterwards 👍